.kokoro/conformance.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -eo pipefail
+
+export NPM_CONFIG_PREFIX=${HOME}/.npm-global
+
+## cd to the parent directory, i.e. the root of the git repo
+cd $(dirname $0)/..
+
+# Build and start the proxy in a separate process
+pushd .
+npm install
+nohup npm run testproxy &
+proxyPID=$!
+popd
+
+# Run the conformance test
+cd cloud-bigtable-clients-test/tests
+eval "go test -v -proxy_addr=:9999 -skip `cat ../../testproxy/known_failures.txt`"
+RETURN_CODE=$?
+
+# Stop the proxy
+kill $proxyPID
```

Comment:
This should be executed irrespective of when / how script exits. Exit traps can be used here. E.g. https://github.com/googleapis/google-cloud-go/blob/5de394be14cf852b99dde65fbeba3d02714f6845/storage/emulator_test.sh#L64-L70

Reply:
Sure. Makes sense. So would you suggest adding `trap kill $proxyPID` somewhere at the top of the file and removing this line? In that case would any exit signal still get propogated up to the user?

Reply:
Add a cleanup function.
Trap on EXIT signal which would catch any exit signal.

Similar to https://github.com/googleapis/google-cloud-go/blob/5de394be14cf852b99dde65fbeba3d02714f6845/storage/emulator_test.sh#L70

Reply:
Done

Reply:
Was the trap change pushed? I don't see the changes here

Reply:
In my git client at about 1:30 I made the changes and pushed them. Not sure why they don't show up.

![image](https://github.com/googleapis/nodejs-bigtable/assets/8935272/5a547bff-d789-4de5-8d8c-f90d8ce79a0b)

Reply:
It is called "Add a cleanup operation"